lib/server/index.js

```diff
@@ -223,6 +227,11 @@ Server.prototype = {
     var config = this.config;
     var test_page = config.get('test_page');
 
+    // If there is an array of test pages, we cycle through them as the default
+    if (Array.isArray(test_page)) {
+      test_page = test_page[this.testPageCount % test_page.length];
+    }
```

Comment:
@stefanpenner I'm not entirely sure I understand what you're getting at.

> a queue of pending segments to run, and available test runners pop off that queue.

It sounds like this is proposing a single, shared queue amongst all test runners, which causes problems if you're expecting every runner (e.g., Phantom and Chrome) to run all pages within the queue. Could you perhaps give an example of what you're proposing?